lib/sim/types.ts

```diff
@@ -15,7 +15,7 @@ export interface SimulationTransientVoltageGraphElement {
   simulation_experiment_id: string
   timestamps_ms?: number[]
   voltage_levels: number[]
-  schematic_voltage_probe_id?: string
+  simulation_voltage_probe_id?: string
```

Comment:
schematic_voltage_probe_id is also used to allow us to e.g. highlight the schematic voltage probe for the graph